contracts/Pool/PoolRegistry.sol

```diff
@@ -216,11 +217,11 @@ contract PoolRegistry is OwnableUpgradeable {
 
         Comptroller implementation = Comptroller(implementationAddress);
 
-        TransparentUpgradeableProxy proxy = new TransparentUpgradeableProxy(
+        BeaconProxy proxy = new BeaconProxy(
             implementationAddress,
-            proxyAdmin,
```

Comment:
Looks like we don't need proxy admin anymore, right?

Reply:
Correct